### DISCUSSION TITLE
Convert model that writen in ColossalAI  to onnx

### DISCUSSION THREAD

**Discussion by nlper-77** [hpcaitech/ColossalAI#170] [Community | Q&A] [answered]
whether support convert pre-trained model that writen in ColossalAI  to onnx?

**Comment by FrankLeeeee** [comment, marked as answer]
Hi, we currently do not support this feature as ONNX is not compatible with the distributed operation in our operators, but we are developing a distributed inference framework if the purpose of conversion to ONNX is to deploy the model.

**Reply by nlper-77** [reply]
Thx!  wait in hope

**Reply by FrankLeeeee** [reply]
No problem! Glad to answer your question.
